### ISSUE TITLE
(Almost) Impossible to select 'Search' cog/filters

### ISSUE THREAD

**Issue by SyCoREAPER** [yuliskov/SmartTube#2210]
I was looking at the changelog about new filters which had me intrigued. 

Tried going to the cog/gear for Search and if 'Show Keyboard Automatically' is selected, it's (almost) impossible to get to.

**What Happens**
You type the search in, you go to refine it but can't. Hitting right just keeps bringing up the Keyboard.

**What's expected to Happen**
'Show Keyboard..' should only show the keyboard if you click on the search bar, not just highlighting it. That prevents you from reaching the filter.

**Workaround**
A temporary workaround I found is to do the search, go into search suggestions right below, scroll all the way to the right (through all of them) and click up. Only way to get to it.

**Details**
Build: 17.20 (but I'm sure this existed way before, I never tried the filter before)
Firecube 3

**Comment by MailYouLater** [comment]
Can't you just close the keyboard with the back button on your remote and then continue to the right to get to it?  This method seems to work fine for me on a MiBox S with Android TV 9.

**Comment by SyCoREAPER** [comment]
No. Not with aforementioned setting enabled. It shifts the "cursor" back to the left.

**Comment by MailYouLater** [comment]
> **Details** Build: 17.20 (but I'm sure this existed way before, I never tried the filter before) Firecube 3

Where did you get 17.20? I didn't think that existed yet...

---
I've tested both the current Stable (16.93) and Beta (17.12) releases, with and without Show Keyboard Automatically enabled and in every case, it functioned as expected <sub><sup>(if I press the back button to close the keyboard, I can press the right button [sometimes multiple times are needed] to exit the the text input field and reach the cog button)</sup></sub> on my MiBox S with Android TV 9.

---
Also, after rereading your post, I think you may have misunderstood something:

> **What's expected to Happen** 'Show Keyboard..' should only show the keyboard if you click on the search bar, not just highlighting it.

Opening the keyboard automatically when the text input gains focus instead of requiring the user to push another button to activate it after entering the input field is literally exactly what that option is for.  That is the one (and only) thing it does.  If you want it to work as you descibed, then... just disable the Show Keyboard Automatically option in the settings.

---
If there's a real problem here, then I can only guess that it's either an incompatibility with another setting, or with Amazon's modifications to Android to make FireOS.  If after reading the above, you still think there's an actual bug ... Can you possibly make a short video which demonstrates the problem?
